apps/src/gamelab/GameLab.js

```diff
@@ -73,7 +73,7 @@ var GameLab = function () {
   this.consoleLogger_ = new JsInterpreterLogger(window.console);
 
   /** @type {JsDebuggerUi} */
-  this.debugger_ = new JsDebuggerUi(this.runButtonClick.bind(this));
+  this.debugger_ = null;
```

Comment:
Why move construction of the `JsDebuggerUi` into init?  It should still be a 'dumb' constructor, allocating a few things but having no DOM side-effects.  That all happens in `initializeAfterDomCreated`.  Building the thing up here will save you a bunch of null-checks.